sdk/cosmosdb/cosmos/samples/v3/javascript/Shared/handleError.js

```diff
@@ -4,9 +4,11 @@
 const path = require("path");
```

Comment:
The `exports` in this file weren't correctly converted by dev-tool. I'll try to run my experimental branch on it and see if it handles it right.